packages/acceptance-tests/pkg-tests-specs/sources/commands/config.test.js

```diff
@@ -0,0 +1,111 @@
+const {
+  fs: {mkdirp, writeFile},
+} = require('pkg-tests-core');
+
+const RC_FILENAME = `.spec-yarnrc`;
+const SUBFOLDER = `subfolder`;
+const FAKE_REGISTRY_URL = `http://yarn.test.registry`;
+const FAKE_WORKSPACE_ROOT = `WORKSPACE_ROOT`;
+
+const environments = {
+  [`folder without rcfile in ancestry`]: async () => {
+    // Nothing to do
+  },
+  [`folder with rcfile`]: async path => {
+    await writeFile(`${path}/${SUBFOLDER}/${SUBFOLDER}/${RC_FILENAME}`, `init-scope berry-test\n`);
+  },
+  [`folder with rcfile without trailing newline`]: async path => {
+    await writeFile(`${path}/${SUBFOLDER}/${SUBFOLDER}/${RC_FILENAME}`, `init-scope berry-test`);
+  },
+  [`folder with rcfile and rc in parent`]: async path => {
+    await writeFile(`${path}/${SUBFOLDER}/${SUBFOLDER}/${RC_FILENAME}`, `init-scope berry-test\n`);
+    await writeFile(`${path}/${SUBFOLDER}/${RC_FILENAME}`, `init-scope berry-test\nlastUpdateCheck 1555784893958\n`);
+  },
+  [`folder with rcfile and rc in ancestor parent`]: async path => {
+    await writeFile(`${path}/${SUBFOLDER}/${SUBFOLDER}/${RC_FILENAME}`, `init-scope berry-test\n`);
+    await writeFile(`${path}/${RC_FILENAME}`, `init-scope berry-test\nlastUpdateCheck 1555784893958\n`);
+  },
+};
+
+function cleanupPlainOutput(output, path) {
+  // replace the generated workspace root with a constant
+  output = output.replace(new RegExp(path, `g`), FAKE_WORKSPACE_ROOT)
+
+  // replace the generated registry server URL with a constant
+  output = output.replace(/^(\s*npmRegistryServer.*?)(['"]?)http:\/\/localhost:\d+\2\s*$/mg, `$1$2${FAKE_REGISTRY_URL}$2`);
+
+  return output;
+}
+
+function cleanupJsonOutput(output, path) {
+  let outputObject;
+  try {
+    outputObject = JSON.parse(output);
+  } catch {
```

Comment:
This syntax is not supported in node 8

```suggestion
  } catch (e) {
```